packages/rx_bloc_cli/example/README.md

```diff
@@ -541,6 +542,16 @@ For more info please visit [widget_toolkit_qr](https://pub.dev/packages/widget_t
 
 On iOS, an issue may occur during the initial build of the application. This can be resolved by opening the `Runner.xcodeproj` file in Xcode and setting the iOS Deployment Target to 16.0 in the Project -> Runner -> Info -> Deployment Target section. Note that this issue only affects the initial build, subsequent builds should succeed without any issues.
 
+
+
+## Feature Onboarding
+
+The onboarding feature introduces an onboarding flow where the user can register their account. 
```

Comment:
It will be great to have a visual representation of each onboarding step. Considering we have golden tests I suppose we can link them in the documentation. The other link we need to add is a link to the API specification for easier access.

Reply:
@StanevPrime  I don't think we have golden tests for the onboarding feature implemented or mentioned as part of the ACs. But we can certainly add them as part of a separate ticket.